Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -586,16 +611,26 @@ - (void)applicationWillTerminate
 - (NSMutableArray *)queue
 {
     if (!_queue) {
+#if TARGET_OS_TV
+        _queue = [[NSUserDefaults standardUserDefaults] objectForKey:SEGQueueKey];
+#else
         _queue = [NSMutableArray arrayWithContentsOfURL:self.queueURL] ?: [[NSMutableArray alloc] init];
+#endif
     }
+
     return _queue;
 }
 
 - (NSMutableDictionary *)traits
 {
     if (!_traits) {
+#if TARGET_OS_TV
+        _traits = [[NSUserDefaults standardUserDefaults] objectForKey:SEGTraitsKey];
```

Comment:
Same as for the queue, this should fallback to ` [[NSMutableDictionary alloc] init]`

`_traits = [[NSUserDefaults standardUserDefaults] objectForKey:SEGTraitsKey]  ?: [[NSMutableDictionary alloc] init];`